lib/package-managers/package-manager.ts

```diff
@@ -1,5 +1,6 @@
 export enum PackageManager {
   NPM = 'npm',
   YARN = 'yarn',
-  PNPM = 'pnpm'
+  PNPM = 'pnpm',
+  NONE = 'none (skip installing packages)',
```

Comment:
I just notice something. If `none` is selected, then npm will be selected due to following lines on `application` schematics

https://github.com/nestjs/schematics/blob/440d06d6217e40ca334be635d5264aac6dda8002/src/lib/application/application.factory.ts#L43-L46

this is needed to generate the `README.md` properly. Thus, wouldn't be better if

 ```suggestion
   NONE = 'none (use NPM but without installing packages)',
```

Reply:
actually, the selected option is not taken in count to generate the files **but** this is bug that was already fixed by #1457, so you could think it is.